src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsrestrictedpatientsapi/gateways/PrisonApiQueryService.kt

```diff
@@ -7,34 +7,14 @@ import org.springframework.web.reactive.function.client.WebClient
 import org.springframework.web.reactive.function.client.WebClientResponseException
 import org.springframework.web.reactive.function.client.bodyToMono
 import reactor.core.publisher.Mono
-import uk.gov.justice.digital.hmpps.hmppsrestrictedpatientsapi.model.entities.RestrictedPatient
-import uk.gov.justice.digital.hmpps.hmppsrestrictedpatientsapi.model.request.CreateExternalMovement
 import uk.gov.justice.digital.hmpps.hmppsrestrictedpatientsapi.model.response.Agency
 import uk.gov.justice.digital.hmpps.hmppsrestrictedpatientsapi.model.response.MovementResponse
 import uk.gov.justice.digital.hmpps.hmppsrestrictedpatientsapi.model.response.OffenderBookingResponse
 
-// Use this to get a WebClient when we are not in a web scope so have no Principal, e.g. for handling events
 @Service
-class PrisonApiApplicationGateway(prisonApiClientCredsAppScope: WebClient) : PrisonApiGateway(prisonApiClientCredsAppScope)
+class PrisonApiQueryService(private val prisonApiSystemClient: WebClient) {
```

Comment:
We have a prison API query service that contains only calls to query endpoints. All authorisation is done via roles so we can always use the system client credentials for these calls.